application/src/main/java/uk/nhs/hee/tis/revalidation/entity/DoctorsForDB.java

```diff
@@ -67,6 +67,7 @@
   private LocalDate lastUpdatedDate;
   private String designatedBodyCode;
   private String admin;
+  private Boolean existsInGmc;
```

Comment:
If this field should "always" default to true, not just for `DoctorsForDb::convert`, then you can initialise it by adding ` = true;` here.